src/components/department/IndexDepartment.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const IndexDepartment = () => {
+  // State to store the departments data
+  const [departments, setDepartments] = useState([]);
+  
+  // State to store the number of departments to display
+  const [displayCount, setDisplayCount] = useState(8);
+
+  // Use effect to fetch the departments data from the API
+  useEffect(() => {
+    fetch('https://openaccess-api.clevelandart.org/api/departments')
```

Comment:
So I understand why you added the fetch url to `https://openaccess-api.clevelandart.org/api/departments`, however when I go to that end point in the browser or in Postman, I get 404 not found. In general just make sure you're looking at the API documentation (or data object) to see what the result returns, because it'll make it easier to target what you're looking for.

Reply:
That said, I'm not sure we need to actually use the API for departments! We talked about seeding the database on this one, and then relating artworks to departments we pre-define. But I totally get why you approached it this way.

Reply:
I believe this means that the api call you want to do is to our backend database. Let's ask Josh tomorrow if he's ready to push up that seed script (I believe he's working on it) so you can use it to render departments.

Reply:
I believe what you'll want to do is 
- [ ] create an API call to our backend at `src/api/departments.js`. Write a function like `getAllDepartments` using the `apiUrl` that points to localhost:8000
- [ ] Import your API call function from the api file to IndexDepartments component
- [ ] Call the function inside useEffect (just as you did above) except this time with the new function

If you need a reference I just took a look at Fruitcake-Pets-Client. Let me know if any of this is unclear, happy to chat more!

Reply:
> I believe this means that the api call you want to do is to our backend database. Let's ask Josh tomorrow if he's ready to push up that seed script (I believe he's working on it) so you can use it to render departments.

>Sounds good! In the meantime I'll work on the things below!